server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2497,7 +2497,7 @@ public Boolean propagateResourceEvent(final long agentId, final ResourceState.Ev
         }
 
         if (s_logger.isDebugEnabled()) {
```

Comment:
Can you remove this IF check ?
I mean, removing only the IF and maintaining the logging.

Reply:
Yeah, they were done

Reply:
Thanks!

Reply:
Thanks!